cmd/auth-rest/startcmd/storage.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package startcmd
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/cenkalti/backoff"
+	"github.com/trustbloc/edge-core/pkg/storage"
+	couchdbstore "github.com/trustbloc/edge-core/pkg/storage/couchdb"
+	"github.com/trustbloc/edge-core/pkg/storage/memstore"
+	"github.com/trustbloc/edge-core/pkg/storage/mysql"
```

Comment:
What's the difference between edge-core mysql and Aries-framework-go-ext mysql?

Reply:
@troyronda hub-auth is not using aries-framework-go.

Reply:
@troyronda edge-core storage has separate CreateStore and OpenStore methods. This was originally done since the EDV server needs the ability to open a store without creating one automatically if it doesn't exist.